pipeline.yml

```diff
@@ -87,6 +87,8 @@ jobs:
         args: ["common-secrets/secrets.yml", "logsearch-manifest/manifest.yml"]
       outputs:
       - name: logsearch-manifest
+      params:
+        LOGSEARCH_INGESTOR_PROFILE: staging-logsearch-ingestor
```

Comment:
Maybe I'm being too lazy here, but I didn't want to add terraform to the pipeline to merge a single, non-sensitive value, especially because hopefully we'll be switching to cloud config soon and doing most of the merging in deploy-bosh. Then again, it's not clear when we'll actually get around to using cloud config. Anyway, if you think it would be better to merge from terraform, I'll update this.